src/main/resources/assets/create/lang/es_es.json

```diff
@@ -1613,7 +1613,7 @@
 	"create.train.status.missing_driver": "El conductor ha desaparecido",
 	"create.train.status.found_driver": "Se ha encontrado un nuevo conductor",
 	"create.train.status.navigation_success": "Navegación exitosa",
-	"create.train.status.no_match": "UNLOCALIZED: No station on graph matches '%1$s'",
+	"create.train.status.no_match": "No station on graph matches '%1$s'",
```

Comment:
The train cannot find the station on the current tracks (not the path to the station, but the station itself)

Reply:
Thanks for both comments, I'll be translating them as soon as I can <3